grande/build.sh

```diff
@@ -30,39 +40,27 @@ function bootstrap {
     # Select default suite
     echo "APT::Default-Release \"$SUITE\";" > "$ROOTFS/etc/apt/apt.conf.d/01defaultrelease"
 
-    # Installing dumb-init
-    curl -o dumb-init.deb -L "$DUMBINIT_URL"
-    dpkg --root "$ROOTFS" -i dumb-init.deb
-
-    cp -r -t "$ROOTFS" "$SCRIPT_DIR"/rootfs/*
-
     # Configure locales
     chroot "$ROOTFS" /usr/sbin/locale-gen
     chroot "$ROOTFS" /usr/sbin/locale-gen en_US.UTF-8
     chroot "$ROOTFS" /usr/sbin/dpkg-reconfigure locales
 
-    echo 'deb http://httpredir.debian.org/debian/ jessie main contrib non-free' > "$ROOTFS/etc/apt/sources.list"
```

Comment:
Maybe make sense to use here `DEBIAN_VERSION` env variable? To not change again in future all these lines below.

Reply:
Do you mean passing a custom variable with that value in via Docker?
Cause I can't find it anywhere and using `/etc/debian_version` seems not ideal, not using `lsb_release` which is not installed

Reply:
Okay, I think I found it.. send a new commit soon